prow/plugins/label/label.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package label
+
+import (
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/plugins"
+
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+const pluginName = "label"
+
+var (
+	allowedPrefixes = []string{"area", "priority"}
+	regexBuilder    = `(?m)^/%s[\t\s]+(?:\S*(?:[^\n]+))*`
```

Comment:
Interestingly, you don't have any capture group. Don't you want to capture the argument list?

`(?:[^\n]+)` doesn't have to be a group at all. Consider replacing with `[^\n]+`

The `%s` is kind of gross, can you replace by the actual command?

I know that you have multiple command, and I see you're having a for loop over the list of prefixes, maybe you should replace directly in the regex?

`/(area|priority) ...`.